### DISCUSSION TITLE
Missing country in Payment Bills

### DISCUSSION THREAD

**Discussion by Jmlpez** [community/community#74548] [GitHub Education]
### Hi everyone,

I'm trying to apply to Github student starter pack(I'm currently studying CS), and  I have to add to my profile the billing information  (I don't have to add any payment method necessarily). The problem is that I'm from Cuba and my country doesn't appear in the available country/regions, and that form field its obligatory.  What can I do?

**Comment by TankEngine-ish** [comment]
I don't know if I remember correctly but few years ago GitHub made certain account restrictions under some US sanctions for countries like Iran, Syria, Cuba etc. I don't know if the student starter pack would be included and don't know if they may have lifted these sanctions. So yeah, your account may only have basic permissions. 

You can contact GitHub Education for support: - https://support.github.com/features/education

**Comment by L1vDev** [comment]
Hi everyone, I have the same problem, any solution about this?

**Comment by TankEngine-ish** [comment]
Hey, 

I don't think the US sanctions (incl. access to GitHub's additional features which come with GitHub Education) have been lifted for Cuba.

**Comment by HotCakeX** [comment]
That's normal, see here for more info: https://www.state.gov/state-sponsors-of-terrorism/

And here: https://orpa.princeton.edu/export-controls/sanctioned-countries

**Comment by L1vDev** [comment]
Well, I don't know how, but i just set my location as default using windows maps app, and then when i go to github education it works as i expected. I guess that it was a problem with my location or something like that.
